systems-ui/src/pages/dashboard/systems-status-overview/SystemsStatusOverview.js

```diff
@@ -0,0 +1,69 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import {VictoryPie} from "victory";
+import styles from "./SystemsStatusOverview.css";
+import classnames from "classnames";
+import {getAllStates, getStateIdentifier} from "../../../common/business/systemsState";
+import {systemType} from "../../../common/types/systemsTypes";
+
+class SystemsStatusOverview extends Component {
```

Comment:
Could also be stateless.

Reply:
Same as above, I will change this to a PureComponent as soon I have some time :)